src/test/unit/controls/Bar/BarLoad-spec.js

```diff
@@ -822,7 +822,7 @@ describe("Bar - Load lifecycle", () => {
             const iconPath = legendItem.querySelector("path");
             expect(iconSVG).not.toBeNull();
             expect(iconSVG.classList).toContain(styles.legendItemIcon);
-            expect(iconSVG.getAttribute("style")).toContain(COLORS.BLUE);
+            expect(iconSVG.getAttribute("style")).toContain(COLORS.GREEN);
```

Comment:
I think i got it. Because of the order of colors changed, we need to update to GREEN here ( 2nd position of color now)